### PR TITLE
Tests + Modified function

### PR DESCRIPTION
**Pull request by spiderjako**
The tests that weren't running were modified with the values they needed and I've also added a functionality to the function assertCanAccessHistoricData_, so that it takes a slug as an argument and doesn't throw an error if the slug is santiment

### CODE REVIEW

File: test/integration/sanRealizedValueTest.js [hunk 1]
```diff
@@ -20,7 +21,7 @@ const {
 describe('SAN_REALIZED_VALUE', () => {
   const expected = { date: 'string', realizedValue: 'number' }
 
-  const response = san.SAN_REALIZED_VALUE(slug, from, to)
+  const response = san.SAN_REALIZED_VALUE(ethereumSlug, from, to)
```
Comment (kalkov) [inline test/integration/sanRealizedValueTest.js]: Why this change was needed?

File: test/integration/sanTokenCirculationTest.js [hunk 2]
```diff
@@ -46,7 +47,7 @@ describe('SAN_TOKEN_CIRCULATION', () => {
   })
 
   it('returns a record per every day', () => {
-    const results = san.SAN_TOKEN_CIRCULATION(slug, from, to)
+    const results = san.SAN_TOKEN_CIRCULATION(ethereumSlug, from, to)
```
Comment (kalkov) [inline test/integration/sanTokenCirculationTest.js]: Why did you changed the slug from santiment to ethereum here?

File: test/utilitiesTest.js [hunk 3]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
   })
 
   it('returns true when period is over HISTORIC_DATA_THRESHOLD', () => {
-    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD)
-
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
```
Comment (kalkov) [inline test/utilitiesTest.js]: You don't need +1 here

File: test/utilitiesTest.js [hunk 4]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
     expect(san.requestedDataIsHistoric_(from)).to.be.true
   })
 })
 
 describe('assertCanAccessHistoricData_', () => {
   it('throws an error when historic data is requested and api key is not present', () => {
-    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD)
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
+    const firstSlug = 'ethereum'
```
Comment (kalkov) [inline test/utilitiesTest.js]: `firstSlug`, `secondSlug` ... are not good names for variables.
Define two consts at the top of the file:
```javascript
const slug = 'santiment'
const ethereumSlug = 'ethereum'
```
and use them in all tests from this file.

File: test/utilitiesTest.js [hunk 5]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
-
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
     expect(san.requestedDataIsHistoric_(from)).to.be.true
   })
 })
 
 describe('assertCanAccessHistoricData_', () => {
   it('throws an error when historic data is requested and api key is not present', () => {
-    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD)
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
```
Comment (kalkov) [inline test/utilitiesTest.js]: +1 is not needed

File: test/utilitiesTest.js [hunk 6]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
+    const firstSlug = 'ethereum'
     expect(() =>
-      san.assertCanAccessHistoricData_(from))
+      san.assertCanAccessHistoricData_(from, firstSlug))
       .to.throw('Full historical data is only accessible to premium users. ' +
                 'Add your API key to use it.')
   })
 
   it("doesn't throw an error when requested data is not historic", () => {
+    const secondSlug = 'ethereum'
```
Comment (kalkov) [inline test/utilitiesTest.js]: remove

File: test/utilitiesTest.js [hunk 7]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
     const from = subDays(now, san.HISTORIC_DATA_THRESHOLD - 1)
-    expect(() => san.assertCanAccessHistoricData_(from)).to.not.throw()
+    expect(() => san.assertCanAccessHistoricData_(from, secondSlug)).to.not.throw()
```
Comment (kalkov) [inline test/utilitiesTest.js]: use `ethereumSlug`

File: test/utilitiesTest.js [hunk 8]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
   })
 
   it("doesn't throw an error when historic data is requested and api key is present", () => {
+    const thirdSlug = 'ethereum'
+    san.setUserProperty_('API_KEY', 'test_api_key')
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
```
Comment (kalkov) [inline test/utilitiesTest.js]: +1 not needed

File: test/utilitiesTest.js [hunk 9]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
 
   it("doesn't throw an error when requested data is not historic", () => {
+    const secondSlug = 'ethereum'
     const from = subDays(now, san.HISTORIC_DATA_THRESHOLD - 1)
-    expect(() => san.assertCanAccessHistoricData_(from)).to.not.throw()
+    expect(() => san.assertCanAccessHistoricData_(from, secondSlug)).to.not.throw()
   })
 
   it("doesn't throw an error when historic data is requested and api key is present", () => {
+    const thirdSlug = 'ethereum'
```
Comment (kalkov) [inline test/utilitiesTest.js]: remove

File: test/utilitiesTest.js [hunk 10]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
+    san.setUserProperty_('API_KEY', 'test_api_key')
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
+    expect(() => san.assertCanAccessHistoricData_(from, thirdSlug)).to.not.throw()
```
Comment (kalkov) [inline test/utilitiesTest.js]: use `ethereumSlug`

File: test/utilitiesTest.js [hunk 11]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
+  })
+  it("doesn't throw an error when the data is historic and no api key is present, but the slug is santiment", () => {
+    const fourthSlug = 'santiment'
```
Comment (kalkov) [inline test/utilitiesTest.js]: remove

File: test/utilitiesTest.js [hunk 12]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
     san.setUserProperty_('API_KEY', 'test_api_key')
-    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD)
-    expect(() => san.assertCanAccessHistoricData_(from)).to.not.throw()
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
```
Comment (kalkov) [inline test/utilitiesTest.js]: +1 not needed

File: test/utilitiesTest.js [hunk 13]
```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
...
+    expect(() => san.assertCanAccessHistoricData_(from, fourthSlug)).to.not.throw()
```
Comment (kalkov) [inline test/utilitiesTest.js]: use `slug`

File: src/functions/socialFunctions.js [hunk 14]
```diff
@@ -40,7 +40,7 @@ function socialVolume_ (slug, from, to, socialVolumeType) {
 }
 
 function trendingWords_ (source, size, hour, from, to) {
-  assertCanAccessHistoricData_(from)
+  assertCanAccessHistoricData_(from, source)
```
Comment (valo) [inline src/functions/socialFunctions.js]: Why is this `source` here? Shouldn't it be `slug`? I guess the problem is that this function here is not working with slugs, so something needs to be passed to it. I think it makes more sense to pass `nil` and indicate an empty slug.
